Logging/ARKLogMessage.h

```diff
@@ -27,15 +27,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ARKLogMessage : NSObject <NSCopying, NSSecureCoding>
 
+/// @warning Do not use this initializer.
+/// @raises NSInternalInconsistencyException This is not a valid initializer.
+- (instancetype)init;
```

Comment:
Purposefully don't want to mark `init` and `new` as unavailable so that subclasses of ARKLogMessage can define init as their designated initializer, if they so wish.